client/packages/common/src/intl/utils/DateUtils.test.ts

```diff
@@ -36,4 +36,10 @@ describe('getDisplayAge', () => {
     const result = getDisplayAge(dob);
     expect(result).toBe('10 days');
   });
+
+  it('returns an empty string if dob is not defined', () => {
+    const dob = null;
+    const result = getDisplayAge(dob);
+    expect(result).toBe('');
+  });
```

Comment:
In simple cases like this I think it's fine to not bother with the intermediate variables, particularly since the description tells you what it's doing. Just this would be okay:
```ts
expect(getDisplayAge(null)).toBe('')
```

It's fine though, no need to change it in this case, just a general comment.